secrets.tf

```diff
@@ -9,7 +9,6 @@ resource "tls_private_key" "gitlab-self-signed-cert-key" {
 }
 
 resource "tls_self_signed_cert" "gitlab-self-signed-cert" {
-  key_algorithm         = "ECDSA"
   private_key_pem       = tls_private_key.gitlab-self-signed-cert-key.private_key_pem
```

Comment:
Why did you remove the algorithm?

Reply:
Not supported anymore

Reply:
ECDSA, it takes it from the 'tls_private_key' resource
https://github.com/SygniaLabs/ScallOps/blob/v0.4/secrets.tf#L7-L8